src/main/resources/hudson/plugins/favorite/user/FavoriteUserProperty/config.jelly

```diff
@@ -1,8 +1,8 @@
 <?jelly escape-by-default='true'?>
 <j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout"
          xmlns:t="/lib/hudson" xmlns:f="/lib/form">
-  <st:include page="/hudson/plugins/favorite/assets/css.jelly" />
-  <st:include page="/hudson/plugins/favorite/assets/javascript.jelly" />
+  <st:include page="/hudson/plugins/favorite/assets/css.jelly" class="${instance.assetClass}" />
```

Comment:
Not the right fix. See [JENKINS-44797](https://issues.jenkins-ci.org/browse/JENKINS-44797).

Reply:
As I wrote there will also be an exception if no class is passed to stapler include:
`Asset.java` : `Just a dummy class for the jelly st:include which otherwise throws exceptions in config.jelly when st:include to css.jelly is called without class parameter`

So maybe there are some breaking changes somewhere in stapler?